src/test/groovy/graphql/execution/preparsed/PreparsedDocumentEntryTest.groovy

```diff
@@ -30,34 +30,58 @@ class PreparsedDocumentEntryTest extends Specification {
         thrown(AssertException)
     }
 
-    def "Ensure a non-null errors returns"() {
+    def "Ensure a non-null error returns"() {
         given:
-        def errors = [new InvalidSyntaxError(new SourceLocation(0, 0), "bang"),
-                      new ValidationError(ValidationErrorType.InvalidSyntax)]
+        def error = new InvalidSyntaxError(new SourceLocation(0, 0), "bang")
 
         when:
-        def docEntry = new PreparsedDocumentEntry(errors)
+        def docEntry = new PreparsedDocumentEntry(error)
```

Comment:
Put this test back (as per above) and write a new one for the single error.  Again for api breaking reasons